.github/workflows/web-docker.yml

```diff
@@ -0,0 +1,119 @@
+name: Web Pipeline
+
+on:
+  push:
+    branches: 
+      - master
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [ master, develop ]
+    types: [ opened, closed ]
+
+env:
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+  ENV: "stg"
+  argocd-file: "values-staging.yaml"
+
+jobs:
+  main:
+    runs-on: ubuntu-latest    
+    name: Web build docker
+    steps:
+      - if: github.ref != 'refs/heads/develop'
+        uses: actions/checkout@v2
+        with:
+          ref: develop
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 100
+
+      - if: github.ref != 'refs/heads/develop'
+        run: git branch develop origin/develop
+
+      - uses: actions/cache@v2
```

Comment:
why do we cache here and not in the ssr?

Reply:
Let me check old workflows , maybe is left over from what I copied from the old workflows